js/main.js

```diff
@@ -0,0 +1,24 @@
+function getRandomNumber(min, max) {
+  if (max <= min || min < 0 || max <= 0) {
+    window.console.log('Неверный диапазон');
+  } else {
+    return Math.floor(Math.random() * (max - min + 1) ) + min;
+  }
+}
+
+// Источник https://schoolsw3.com/js/js_random.php
```

Comment:
```suggestion
// Источник https://schoolsw3.com/js/js_random.php
function getRandomNumber(min, max) {
  if (max <= min || min < 0 || max <= 0) {
    return console.log('Неверный диапазон');
  } 
  
  return Math.floor(Math.random() * (max - min + 1) ) + min;
}
```

1. Комментарии пишутся перед функциями, а не после.
2. если в иф и элсе происходит ретерн, то можно блок элсе опустить
3. для доступа к console.log не обязательно объявлять window
4. Вообще консоль лог плохо использовать, лучше ошибку выбросить